script/dev/hook_setup.sh

```diff
@@ -0,0 +1,14 @@
+echo "< ========= Setup Start ========= >"
```

Comment:
This seems nice, but difficult to maintain. Maybe we can use something as Husky instead?
https://github.com/typicode/husky

Reply:
You can also use lint-staged, which is the easiest one. Just pass the command to lint-staged to run then make husky call it

Reply:
lint-staged seems to work only for pre-commit hooks. I'm skipping lint-staged, as my requirement is for pre-push hook.